src/components/Profile/OnClickTextEdit.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
So this looks good and seems to be working fine.  My one concern is that there doesn't seem to be a way to submit/save changes.  We need all this info to be sent to the database when the user taps a button like "save changes" or something like that.  It might be easiest to, rather than having all these little self-editing fields, have a button at the bottom of the profile page (maybe that little icon of a pen) and when the user taps it, they enter edit mode and can edit everything and the little edit icon disappears and is replaced by a "save" button and a "cancel" button.  Then when you press save, the information updates our DB and when you press "cancel" everything goes back to the way it was before.